machine_management/control-plane-machine-management.adoc

```diff
@@ -0,0 +1,103 @@
+:_content-type: ASSEMBLY
+[id="control-plane-machine-management"]
+= Managing control plane machines
+include::_attributes/common-attributes.adoc[]
+:context: control-plane-machine-management
+
+toc::[]
+
+With the Control Plane Machine Set Operator, you can automate management of the control plane machine resources within your {product-title} cluster. 
+
+[IMPORTANT]
+====
+Control plane machine sets cannot manage compute machines, and compute machine sets cannot manage control plane machines.
+====
+
+//Trying to balance the two being analogous (so, familiar to admins), but not the same (so, not conflated by admins). I think this language needs a lot of careful scrutiny and consideration.
+Control plane machine sets provide for control plane machines similar management capabilities as compute machine sets provide for compute machines. However, these two types of machine sets are separate custom resources defined within the Machine API and have several fundamental differences in their architecture and functionality.
+
+{product-title} clusters that are created with version 4.12 or later use the Control Plane Machine Set Operator by default. To use this Operator on clusters that are upgraded to version 4.12 or later from an earlier version, you must xref:../machine_management/control-plane-machine-management.adoc#cpmso-installing_control-plane-machine-management[install the Operator].
```

Comment:
At the moment, it's manual install on Azure, auto install on AWS, manual on platforms that do not support failure domains (eg baremetal, vsphere). Not supported on GCP. Somewhere we will need a grid to clarify all of this